### ISSUE TITLE
Training data requirement

### ISSUE THREAD

**Issue by farooqkbs** [chen0040/keras-english-resume-parser-and-analyzer#7]
Hello there

great work, just would like to know how much training data would be sufficient to train the model and if there is already training data repo available that I could use for my project?

Thanks in Advance

Farooq

**Comment by A-Jatin** [comment]
I have the same query. Following.

**Comment by saipavanmeruga** [comment]
+1

**Comment by maheshpj** [comment]
+1

**Comment by meedeepak** [comment]
+1

**Comment by sumit-gupta-sgt** [comment]
I am trying to generate Training Data but once form pop up, i am not able to see submit to generate Training Data, Anybody able to generate training data?

**Comment by sayalraza** [comment]
+1

**Comment by aaadarshbisht** [comment]
Any Updates?

**Comment by haaks1998** [comment]
Does anyone found dataset to train it???

**Comment by ylmz-dev** [comment]
Any Updates?

**Comment by iamghanti** [comment]
you need to first evaluate by some resumes. before you hit the training repo. am creating one. once i have i will share

**Comment by Creative-Priyamvada** [comment]
@iamghanti if you have created it, please do share. Will highly appreciate it

**Comment by GOVINDBHAVAN** [comment]
> you need to first evaluate by some resumes. before you hit the training repo. am creating one. once i have i will share

Please share the training repo, if you have prepared a sample. Thanks in advance.

**Comment by aryansetia** [comment]
> > you need to first evaluate by some resumes. before you hit the training repo. am creating one. once i have i will share
> 
> Please share the training repo, if you have prepared a sample. Thanks in advance.

Hi, is the model working for you?

**Comment by iamghanti** [comment]
since the data which I prepared are confidential I can't share all, check sample attached once you generate like this format(you should be also able to prepare this using AnnotatorGui) start experimenting with adding more meta fields, my accuracy dropped after adding more fields
[0.txt](https://github.com/chen0040/keras-english-resume-parser-and-analyzer/files/8523529/0.txt)
[1.txt](https://github.com/chen0040/keras-english-resume-parser-and-analyzer/files/8523530/1.txt)

**Comment by aryansetia** [comment]
> since the data which I prepared are confidential I can't share all, check sample attached once you generate like this format(you should be also able to prepare this using AnnotatorGui) start experimenting with adding more meta fields, my accuracy dropped after adding more fields [0.txt](https://github.com/chen0040/keras-english-resume-parser-and-analyzer/files/8523529/0.txt) [1.txt](https://github.com/chen0040/keras-english-resume-parser-and-analyzer/files/8523530/1.txt)

Can we connect on Linkedin?
I wanted to ask few things

**Comment by GOVINDBHAVAN** [comment]
> 

Thanks for the reply.
Can you please also brief on difference between header, meta and content.

Person technical skills should we tag as meta or content.
What is difference between meta and content.
